### ISSUE TITLE
Administative Organisations-Dashboard

### ISSUE THREAD

**Issue by datenbrei** [Human-Connection/Human-Connection#278]
As organisation support I'd like to have an overview about the status and profile of every organisation in order to keep control of the onboarding process of organisations and unlock a specific organisation's profile.

**Comment by mattwr18** [comment]
still waiting for clarification... duplicate of https://github.com/Human-Connection/Human-Connection/issues/418 ??
blocked by https://github.com/Human-Connection/Human-Connection/issues/281
